### PR TITLE
Add terraform-docs

### PR DESCRIPTION
**Pull request by Dirrk**
terraform-docs:  https://github.com/segmentio/terraform-docs
terraform-docs action: https://github.com/Dirrk/terraform-docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,6 +197,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Edit JSON File](https://github.com/deef0000dragon1/json-edit-action)
 - [Build Slate documentation](https://github.com/Decathlon/slate-builder-action)
 - [Get latest SemVer and branch name given a search string](https://github.com/jessicalostinspace/github-action-get-regex-branch)
+- [Generate terraform documentation](https://github.com/Dirrk/terraform-docs) - Uses terraform-docs to generate docs for terraform modules
```
Comment (gep13) [inline README.md]:
```suggestion
- [Generate terraform documentation](https://github.com/Dirrk/terraform-docs) - Uses terraform-docs to generate docs for terraform modules.
```

**Comment by gep13** [comment]
@Dirrk can you make the small suggested change, so that the linter passes its checks? Thanks

**Comment by Dirrk** [comment]
@gep13 made the requested change + merged in the upstream

**Comment by gep13** [comment]
@Dirrk your changes have been merged, thanks for your contribution 👍

**Pull request merged**